.github/workflows/main.yml

```diff
@@ -6,9 +6,7 @@ name: Run tests
 on:
   # Triggers the workflow on push or pull request events but only for the master branch
   push:
-    branches: 
-      - master
-      - stripe-endpoint
```

Comment:
FYI if you want to run tests on a branch, open a PR for it (possibly WIP one)

Reply:
I thought that should work - I was probably doing something else wrong. =)